client/catalog/src/app/util.js

```diff
@@ -0,0 +1,14 @@
+export let cap= (s) => {
+	if (!s) {
+		return "";
+	}
+	s = s.replace(/[\w]([A-Z])/g, function(m) {
+           return m[0] + " " + m[1];
+       });
+	return s.charAt(0).toUpperCase() + s.slice(1)
+}
+
+export let restricts= (s) => {
```

Comment:
maybe rename to "restrictToIdent" or something more precise if it's now out of context